$NetBSD: patch-data_meson.build,v 1.1 2022/02/18 17:42:34 pgoyette Exp $

https://gitlab.gnome.org/GNOME/gnome-sudoku/-/commit/7c9935a02b48f332f67bad3e4ee020b75591084a
https://bugs.gentoo.org/831556

From: Jan Beich <jbeich@FreeBSD.org>
Date: Mon, 24 Jan 2022 11:44:06 +0000
Subject: [PATCH] meson: drop unused argument for i18n.merge_file()

Ignored in Meson < 0.60.0, deprecated since 0.60.1 and fatal since 0.61.0.

data/meson.build:5:0: ERROR: Function does not take positional arguments.
data/meson.build:24:0: ERROR: Function does not take positional arguments.

--- data/meson.build.orig
+++ data/meson.build
@@ -2,7 +2,7 @@ resource_files = files('gnome-sudoku.gresource.xml')
 
 resources = gnome.compile_resources('gnome-sudoku', resource_files)
 
-desktop_file = i18n.merge_file('desktop',
+desktop_file = i18n.merge_file(
   input: '@0@.desktop.in'.format(application_id),
   output: '@0@.desktop'.format(application_id),
   install: true,
@@ -21,7 +21,7 @@ if desktop_file_validate.found()
   )
 endif
 
-appdata_file = i18n.merge_file('appdata',
+appdata_file = i18n.merge_file(
   input: '@0@.appdata.xml.in'.format(application_id),
   output: '@0@.appdata.xml'.format(application_id),
   install: true,
